common/build.gradle

```diff
@@ -144,16 +145,16 @@ android.libraryVariants.all { variant ->
 // Create tasks to generate javadoc jar
 android.libraryVariants.all { variant ->
     task("${variant.name}JavadocJar", type: Jar, dependsOn: "${variant.name}Javadoc") {
-        classifier = 'javadoc'
-        from tasks["${variant.name}Javadoc"].destinationDir
+        archiveClassifier.set("javadoc")
+        from tasks["${variant.name}Javadoc"].destinationDir // 'destinationDir' is still reqd here.
     }
 }
 
```

Comment:
You probably want a spot bugs configuration to set it to accept failure, and a backlog task to fix the failures.

Reply:
Have to do this in a few places:
- PMD: 676190a02fe33c1acd84c930fe8c6d439e7c09ae
- SpotBugs: f7eb3b5e53378d00e1498a1b7509072dadb33aeb

Reply:
TODO: Create backlog items